src/api/app/assets/stylesheets/webui/cm2/bootstrap.scss

```
@@ -1,46 +1,59 @@
 // Bootstrap theme for CodeMirror
 .cm-s-bootstrap {
+  --cm-body: var(--bs-body-color);
+  --cm-blue: #{$blue};
+  --cm-pink: #{$pink};
+  --cm-indigo: #{$indigo};
+  --cm-teal: #{$teal};
+  --cm-orange: #{$orange};
+  --cm-red: #{$danger};
+  --cm-gray: #{$text-muted};
+  --cm-purple: #{$purple};
+  --cm-yellow: #{$warning};
+  --cm-green: #{$success};
```

```suggestion
  --cm-red: #{$red};
  --cm-gray: #{$text-muted};
  --cm-purple: #{$purple};
  --cm-yellow: #{$yellow};
  --cm-green: #{$green};
```
Does it make sense to adapt these colors to the theme color scheme? Shouldn't they be fixed colors?